tests/unit/test___init__.py

```diff
@@ -0,0 +1,36 @@
+import pytest
+from _finetuner.excepts import SelectModelRequired
+from _finetuner.models.inference import ONNXRuntimeInferenceEngine, TorchInferenceEngine
+
+import finetuner
+
+
+@pytest.mark.parametrize(
+    'descriptor, select_model, is_onnx, expect_error',
+    [
+        ('bert-base-cased', None, False, None),
+        ('bert-base-cased', None, True, None),
+        ('openai/clip-vit-base-patch16', 'clip-text', False, None),
+        ('openai/clip-vit-base-patch16', 'clip-vision', False, None),
+        ('openai/clip-vit-base-patch16', None, False, SelectModelRequired),
+        ('MADE UP MODEL', None, False, ValueError),
+    ],
+)
+def test_build_model_and_embed(descriptor, select_model, is_onnx, expect_error):
+
+    if expect_error:
+        with pytest.raises(expect_error):
+            model = finetuner.build_model(
+                name=descriptor,
+                select_model=select_model,
+                is_onnx=is_onnx,
+            )
+    else:
+        model = finetuner.build_model(
+            name=descriptor, select_model=select_model, is_onnx=is_onnx
+        )
```

Comment:
if build_model returns nothing it would actually pass. Testing something with the model would be better. Maybe you can just check if you can pass an input and the result is a vector of the expected shape. If this is too much, you could at least check if the type of `model` is correct and not None